.github/workflows/ami-build-ubuntu-18.yml

```diff
@@ -46,6 +46,8 @@ jobs:
       - uses: docker/setup-buildx-action@v2
         with:
           endpoint: builders          
+          driver-opts: image=moby/buildkit:v0.11.6
```

Comment:
Updated buildkit just in case it'd help us avoid connection reset errors; default was running under v0.10.0